### PR TITLE
feat(footer): Hackathon egg for learn page footer

### PR DESCRIPTION
**Pull request by borisonr**
### Overview

When learners click on specific words in the footer it'll open cute p5 visualizations

[integration pr](https://github.com/codecademy-engineering/Codecademy/pull/23488)

### PR Checklist

- [ ] Related to designs:
- [ ] Related to JIRA ticket: ABC-123
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.70-alpha.9edf84.0<br/>@codecademy/gamut-labs@15.2.10-alpha.9edf84.0<br/>@codecademy/styleguide@36.11.2-alpha.9edf84.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60905ca5a7ea65009d2d9723--gamut-preview.netlify.app>https://60905ca5a7ea65009d2d9723--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60905ca5a7ea65009d2d9723>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.70-alpha.5db6a4.0<br/>@codecademy/gamut-labs@15.2.10-alpha.5db6a4.0<br/>@codecademy/styleguide@36.11.2-alpha.5db6a4.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6090621def5db1008dc2c516--gamut-preview.netlify.app>https://6090621def5db1008dc2c516--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6090621def5db1008dc2c516>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.70-alpha.0a1c82.0<br/>@codecademy/gamut-labs@15.2.10-alpha.0a1c82.0<br/>@codecademy/styleguide@36.11.2-alpha.0a1c82.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6090661fa8c59e009f8e762d--gamut-preview.netlify.app>https://6090661fa8c59e009f8e762d--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6090661fa8c59e009f8e762d>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.71-alpha.85da07.0<br/>@codecademy/gamut-labs@15.2.11-alpha.85da07.0<br/>@codecademy/styleguide@36.11.3-alpha.85da07.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://609152fa155aec21a0cc981d--gamut-preview.netlify.app>https://609152fa155aec21a0cc981d--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/609152fa155aec21a0cc981d>Deploy Logs</a></p>

File: packages/gamut-labs/src/GlobalFooter/FooterLegal/MadeIn.tsx [hunk 1]
```diff
@@ -8,17 +8,46 @@ const HeartContainer = styled.span`
   margin-right: 0.25rem;
   min-width: 1.75em;
   text-align: center;
+  cursor: pointer;
 `;
 
-export const MadeIn = () => {
+const ClickableSpan = styled.span`
+  cursor: pointer;
+`;
```
Comment (JoshuaKGoldberg) [inline packages/gamut-labs/src/GlobalFooter/FooterLegal/MadeIn.tsx:16]: Hmm. Is it possible to use one of the new `Anchor` variants for this instead? That's what they're made for.

**Review by JoshuaKGoldberg — CHANGES_REQUESTED** [review]
Unit tests please? At least in this level I think it should be unit testable?

File: packages/gamut-labs/src/GlobalFooter/FooterLegal/MadeIn.tsx (hunk 1 above)
Comment (codecaaron) [reply packages/gamut-labs/src/GlobalFooter/FooterLegal/MadeIn.tsx:16]: `<Anchor variant="interface" />` should work (orrrr we the new emotion `ButtonBase` might do the trick)

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.79-alpha.c0a644.0<br/>@codecademy/gamut-labs@15.5.3-alpha.c0a644.0<br/>@codecademy/styleguide@36.14.1-alpha.c0a644.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6092d2c3e6d3762a8423bc41--gamut-preview.netlify.app>https://6092d2c3e6d3762a8423bc41--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6092d2c3e6d3762a8423bc41>Deploy Logs</a></p>

File: packages/gamut-labs/src/GlobalFooter/FooterLegal/__tests__/MadeIn-test.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { setupRtl } from '@codecademy/gamut-tests';
+import { fireEvent } from '@testing-library/dom';
+
+import { MadeIn } from '../MadeIn';
+
+const renderView = setupRtl(MadeIn, {
+  onClick: jest.fn(),
+});
+
+describe('MadeIn', () => {
```
Comment (borisonr) [inline packages/gamut-labs/src/GlobalFooter/FooterLegal/__tests__/MadeIn-test.tsx:10]: apologies for being lazy on testing, just wanted to get this through :)
was there anything in particular you wanted tested @JoshuaKGoldberg ?

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.80-alpha.88b104.0<br/>@codecademy/gamut-labs@15.5.4-alpha.88b104.0<br/>@codecademy/styleguide@36.14.2-alpha.88b104.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6092d54d09811a00d116026f--gamut-preview.netlify.app>https://6092d54d09811a00d116026f--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6092d54d09811a00d116026f>Deploy Logs</a></p>

File: packages/gamut-labs/src/GlobalFooter/FooterLegal/index.tsx [hunk 3]
```diff
@@ -22,14 +22,18 @@ const LegalLocalContainer = styled.div`
 
 export type FooterLegalProps = {
   onClick: GlobalFooterClickHandler;
+  onMadeInClick?: (e: React.MouseEvent | React.KeyboardEvent) => void;
```
Comment (JoshuaKGoldberg) [inline packages/gamut-labs/src/GlobalFooter/FooterLegal/index.tsx:25]: Recapping Slack convo: https://github.com/codecademy-engineering/Codecademy/pull/23488#discussion_r625835878 only reads from the event target, so maybe just pass `"made" | "in" ...`? (or `string` if you don't particularly care what the values are?)

File: packages/gamut-labs/src/GlobalFooter/FooterLegal/__tests__/MadeIn-test.tsx (hunk 2 above)
Comment (JoshuaKGoldberg) [reply packages/gamut-labs/src/GlobalFooter/FooterLegal/__tests__/MadeIn-test.tsx:10]: LGTM 😄  thanks!

**Review by JoshuaKGoldberg — APPROVED** [review]

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.80-alpha.964bd8.0<br/>@codecademy/gamut-labs@15.5.4-alpha.964bd8.0<br/>@codecademy/styleguide@36.14.2-alpha.964bd8.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6092e3f8ddcaf60f1cb125e5--gamut-preview.netlify.app>https://6092e3f8ddcaf60f1cb125e5--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6092e3f8ddcaf60f1cb125e5>Deploy Logs</a></p>

File: packages/gamut-labs/src/GlobalFooter/FooterLegal/MadeIn.tsx [hunk 4]
```diff
@@ -1,24 +1,46 @@
...
+      </Anchor>{' '}
+      with
+      <HeartContainer variant="interface" onClick={onHeartClick}>
+        ️❤️
+      </HeartContainer>
+      in{' '}
+      <Anchor variant="interface" onClick={onNYCClick}>
+        NYC
+      </Anchor>{' '}
+      ©{` ${new Date().getFullYear()} `}{' '}
```
Comment (JoshuaKGoldberg) [inline packages/gamut-labs/src/GlobalFooter/FooterLegal/MadeIn.tsx:40]:
```suggestion
      ©{` ${new Date().getFullYear()} `}
```

**Review by JoshuaKGoldberg — APPROVED** [review]
Great!

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.80-alpha.1009e4.0<br/>@codecademy/gamut-labs@15.5.4-alpha.1009e4.0<br/>@codecademy/styleguide@36.14.2-alpha.1009e4.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6092f50a940b441307b80867--gamut-preview.netlify.app>https://6092f50a940b441307b80867--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6092f50a940b441307b80867>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.81-alpha.af531c.0<br/>@codecademy/gamut-labs@15.5.5-alpha.af531c.0<br/>@codecademy/styleguide@36.14.3-alpha.af531c.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60930aeb0e482e257850bc16--gamut-preview.netlify.app>https://60930aeb0e482e257850bc16--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60930aeb0e482e257850bc16>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.83-alpha.6a0a92.0<br/>@codecademy/gamut-labs@15.5.7-alpha.6a0a92.0<br/>@codecademy/styleguide@37.0.1-alpha.6a0a92.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6093f8e7a111a500cd20c339--gamut-preview.netlify.app>https://6093f8e7a111a500cd20c339--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6093f8e7a111a500cd20c339>Deploy Logs</a></p>

**Pull request merged**